### PR TITLE
add preferred route and target asset param

### PR DESCRIPTION
**Pull request by sebastianscatularo**
image

### CODE REVIEW

**Review by yuli-ferna — APPROVED** [review]

File: apps/docs/docs/tutorials/how-to-deep-link.md [hunk 1]
```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
 ---
 # Deep link to Portal Bridge
 
+> Note: this feature could work differently between the different bridged experiences
+> that currently portal bridge offers. The documentation presented here works on https://portalbridge.com
```
Comment (mliu-xlabs) [inline apps/docs/docs/tutorials/how-to-deep-link.md:7]: Rewrite this as follows: Note: This feature could work differently between the different bridge versions than what Portal Bridge currently offers. The documentation presented here works on https://portalbridge.com

File: apps/docs/docs/tutorials/how-to-deep-link.md [hunk 2]
```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
...
+
+## Pre-select Assets and Preferred route
+
+|Query Param  | Description |
+|---|---|---|
+|`sourceChain`                   |A source chain that will be pre-selected |
+|`targetChain`                   |A target chain that will be pre-selected |
+|`sourceAsset` or `asset`        |It will be the key of an asset that is injected in configs, you could check [here](https://github.com/XLabs/portal-bridge-ui/blob/main/apps/connect/src/env/token-bridge.mainnet.ts#L505)|
```
Comment (mliu-xlabs) [inline apps/docs/docs/tutorials/how-to-deep-link.md:15]: just double checking that its 'asset' and not 'Asset'

File: apps/docs/docs/tutorials/how-to-deep-link.md [hunk 3]
```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
...
+- to prefer CCTP Manual, the value must be `ManualCCTP`
+- to prefer CCTP Automatic, the value must be `AutomaticCCTP`
+
+##### Token Bridge
+- to prefer Token Bridge Manual, the value must be `ManualTokenBridge`
+- to prefer Token Bridge Automatic, the value must be `AutomaticTokenBridge`
+
+
+##### NTT
+- to prefer Token Bridge Manual, the value must be `ManualNtt`
```
Comment (mliu-xlabs) [inline apps/docs/docs/tutorials/how-to-deep-link.md:40]: to prefer NTT Manual route

File: apps/docs/docs/tutorials/how-to-deep-link.md [hunk 4]
```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
...
+- to prefer Token Bridge Automatic, the value must be `AutomaticNtt`
```
Comment (mliu-xlabs) [inline apps/docs/docs/tutorials/how-to-deep-link.md:41]: to prefer NTT Automatic route

File: apps/docs/docs/tutorials/how-to-deep-link.md [hunk 5]
```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
 ---
 # Deep link to Portal Bridge
 
+> Note: this feature could work differently between the different bridged experiences
+> that currently portal bridge offers. The documentation presented here works on https://portalbridge.com
+
+## Pre-select Assets and Preferred route
+
+|Query Param  | Description |
```
Comment (mliu-xlabs) [inline apps/docs/docs/tutorials/how-to-deep-link.md:11]: Query Param -> Query Parameter

File: apps/docs/docs/tutorials/how-to-deep-link.md [hunk 6]
```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
...
+> it will fall back to the fastest available option.
+
+##### Mayan
+- to prefer Mayan route, the value must be `MayanSwapWH`
+- to prefer Mayan Swift route, the value must be `MayanSwapSWIFT`
+- to prefer Mayan MCTP route, the value must be `MayanSwapMCTP`
+
+
+##### CCTP
+- to prefer CCTP Manual, the value must be `ManualCCTP`
```
Comment (mliu-xlabs) [inline apps/docs/docs/tutorials/how-to-deep-link.md:31]: CCTP Manual route

File: apps/docs/docs/tutorials/how-to-deep-link.md [hunk 7]
```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
...
+- to prefer CCTP Automatic, the value must be `AutomaticCCTP`
```
Comment (mliu-xlabs) [inline apps/docs/docs/tutorials/how-to-deep-link.md:32]: CCTP Automatic route

File: apps/docs/docs/tutorials/how-to-deep-link.md [hunk 8]
```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
...
+
+##### Token Bridge
+- to prefer Token Bridge Manual, the value must be `ManualTokenBridge`
```
Comment (mliu-xlabs) [inline apps/docs/docs/tutorials/how-to-deep-link.md:35]: Token Bridge Manual route

File: apps/docs/docs/tutorials/how-to-deep-link.md [hunk 9]
```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
...
+- to prefer Token Bridge Automatic, the value must be `AutomaticTokenBridge`
```
Comment (mliu-xlabs) [inline apps/docs/docs/tutorials/how-to-deep-link.md:36]: Token Bridge Automatic route

**Review by mliu-xlabs — CHANGES_REQUESTED** [review]

File: apps/docs/docs/tutorials/how-to-deep-link.md (hunk 2 above)
Comment (sebastianscatularo) [reply apps/docs/docs/tutorials/how-to-deep-link.md:15]: yes, it is asset, and it is case sensitive

**Pull request merged**